packages/stores/types.ts

```diff
@@ -103,6 +103,7 @@ export type TRootStore = {
     client: TClientStore;
     common: TCommonStore;
     ui: TUiStore;
+    modules: Record<string, any>;
```

Comment:
```suggestion
    modules: Record<string, unknown>;
```
Should be `unknown` to force the developers to do a type check, But since right now `CFDStore` has been used in the `@deriv/appstore` this is not possible to do.